demo/admin/src/dam/ImportFromUnsplash.tsx

```diff
@@ -0,0 +1,85 @@
+import { useApolloClient } from "@apollo/client";
+import { CancelButton, messages, SaveButton } from "@comet/admin";
+import { useDamAcceptedMimeTypes } from "@comet/cms-admin";
+import { useFileUpload } from "@comet/cms-admin/lib/dam/DataGrid/fileUpload/useFileUpload";
+import { clearDamItemCache } from "@comet/cms-admin/lib/dam/helpers/clearDamItemCache";
```

Comment:
You're using two private APIs, which is not allowed.

`useFileUpload` probably needs the be part of the public API. We should rename it to `useDamFileUpload` to be clear that this is related to the DAM.

`clearDamItemCache` shouldn't be part of the public API. We could remove the need for it by always running the `onAfterUpload` callback. @thomasdax98 why do we need this callback at all?

Reply:
I'm actually not sure why we need it. Since we always do the same thing in `onAfterUpload()` we could move the current `onAfterUpload()` into `useFileUpload()`. Then there is no need to set it from the outside.